src/desktop/apps/artwork/templates/index.jade

```diff
@@ -8,32 +8,37 @@ block head
 
 block body
   - isAuction = artwork.context && artwork.context.__typename == 'ArtworkContextAuction'
-  
+  - isEcommerceSale = artwork.sale && artwork.sale.is_auction == false
+
   if isAuction
     .artwork-page.main-layout-container
       section.artwork
         .artwork__main
           .artwork__main__overview
-            include ../components/banner/index
+            unless isEcommerceSale
```

Comment:
I wonder if you need this in this branch, since `isEcommerceSale` includes `is_auction == false`, but then this branch is under `if isAuction`.

Reply:
Buy now sales are somewhat weird right now; as I understand it, even though something isn't an auction it can still be a sale, and if `is_auction` is false under a sale, then it's an e-commerce artwork. This is the property that we use in a few different places to conditionally render things. That said, I didn't need the `unless` in the top branch, since if its in an auction, it definitely isn't a buy now artwork.